src/Room.java

```diff
@@ -98,36 +102,29 @@ int getNumberOfOnePersonaBeds(){
         return number;
     }
 
-    boolean IsAPerfectMatchRoom(int numberOfCouples, int numberOfPeopleSleepingAlone){
-        int numberOfCouplesCount = 0;
-        int numberOfPeopleSleepingAloneCount = 0;
+    boolean IsAPerfectMatchRoom(int personas){
+        int numberOfPersonasCount = 0;
         for (AbstractCommodity s : this.commoditySet) {
-            if(s.getNumberOfPersonasPerBed()==2) {
-                numberOfCouplesCount++;
-            }
-            if(s.getNumberOfPersonasPerBed()==2) {
-                numberOfPeopleSleepingAloneCount++;
+            if(s.getNumberOfPersonasPerBed()>0) {
+                numberOfPersonasCount++;
             }
         }
-        if(numberOfCouplesCount==numberOfCouples && numberOfPeopleSleepingAloneCount==numberOfPeopleSleepingAlone){
+        if(numberOfPersonasCount==personas){
             return true;
         }
         else return false;
     }
 
-    boolean bookRoomForInterval(String from, String to, String guestName, String guestId) throws ParseException {
-        int sizeBookings = this.bookings.size();
-        Date dateFrom = new SimpleDateFormat("yyyy/mm/dd").parse(from);
-        Date dateTo = new SimpleDateFormat("yyyy/mm/dd").parse(to);
-        Booking booking = new Booking();
-        booking.Booking(guestName, guestId, dateFrom, dateTo);
-        this.bookings.add(booking);
-        if(sizeBookings==this.bookings.size()){
+    boolean bookRoomForInterval(String stringFrom, String stringTo, String guestName, String guestId) {
```

Comment:
Javadoc and access modifier.

Wouldn't it be more clean if you use Date instead of String and parsing everytime?